iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/infospace/converter/PersonConverterTest.java

```diff
@@ -0,0 +1,82 @@
+package eu.dnetlib.iis.wf.importer.infospace.converter;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+import eu.dnetlib.data.proto.OafProtos.OafEntity;
+import eu.dnetlib.data.proto.OafProtos.OafEntity.Builder;
+import eu.dnetlib.data.proto.PersonProtos.Person.Metadata;
+import eu.dnetlib.data.proto.TypeProtos.Type;
+import eu.dnetlib.iis.importer.schemas.Person;
+
+/**
+ * Tests for {@link PersonConverter}
+ */
+public class PersonConverterTest {
+
+    private static final String PERSON_ID = "some id";
+    private static final String FIRST_NAME = "a first name";
+    private static final String SECOND_NAME = "a second name";
+    private static final String SECOND_SECOND_NAME = "another name";
+    private static final String FULL_NAME = "the full name";
+
+    private PersonConverter converter = new PersonConverter();
+
+    //------------------------ TESTS --------------------------
+
+    @Test(expected=NullPointerException.class)
+    public void convert_null_oafEntity() {
+
+        // execute
+        converter.convert(null);
+    }
+
+    @Test
+    public void convert_unset_person() {
+        // given
+        OafEntity oafEntity = emptyEntityBuilder(PERSON_ID).build();
+
+        // execute
+        Person person = converter.convert(oafEntity);
+
+        // assert
+        assertEquals(PERSON_ID, person.getId());
+        assertNull(person.getFirstname());
+        assertNull(person.getSecondnames());
+        assertNull(person.getFullname());
```

Comment:
OK. So the test will have to change after fixing that bug. But for now it is what it is.